index.html

```diff
@@ -218,12 +218,11 @@ <h3>ShipMonk AWS infrastructure</h3>
                              data-animate-effect="fadeIn">
 
                             <figure>
-                                <img src="images/logos/signageos-case-study-cover.png" alt="signageOS case study cover"
+                                <img src="images/logos/keboola-case-study-cover.png" alt="keboola case study cover"
                                      class="img-responsive">
                             </figure>
-
-                            <h3>signageOS AWS infrastructure</h3>
-                            <p>Instead of internally building and operating its own infrastructure, signageOS can leverage managed services from AWS, without worrying much about scalability and maintenance.</p>
+                            <h3>Keboola Elastic Kubernetes Service</h3>
+                            <p>Using a combination of Elastic Kubernetes Service with Managed Nodes, Autoscaling groups and Spot Instances, helps Keboola to manage their cloud cost and pay only for what they truly use. </p>
```

Comment:
why is there a trailing space there? :D 
```
use. </p>
```

Reply:
got rid of 'em!
And cheged the case study pic. Better now maybe?

Reply:
noice chege 🇨🇺